lib/anyplayer/selector.rb

```diff
@@ -22,24 +23,21 @@ def initialize
     # Returns an instance of the first music player that's launched
     def player
       PLAYERS.each do |player|
-        player_load(player) or next
+        player_load(player) || next
 
         instance = player_class(player).new
-        player_on_platform?(instance) or next
+        player_on_platform?(instance) || next
 
-        if player_launched?(instance)
-          return instance
-        end
+        return instance if player_launched?(instance)
       end
       nil
     end
 
-
     private
 
     def platform
       @platform ||= begin
-        host_os = RbConfig::CONFIG["host_os"]
+        host_os = RbConfig::CONFIG['host_os']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.